### PR TITLE
Update OpenApi config to use "security" instead of "x-security".

### PR DESCRIPTION
**Pull request by liminw**

### CODE REVIEW

File: endpoints/getting-started/openapi.yaml [hunk 1]
```diff
@@ -106,4 +101,7 @@ securityDefinitions:
     flow: "implicit"
     type: "oauth2"
     x-google-issuer: "https://accounts.google.com"
-    x-google-jwks_uri: "https://www.googleapis.com/oauth2/v1/certs"
+    x-google-jwks_uri: "https://www.googleapis.com/oauth2/v3/certs"
+    # Your OAuth2 client's Client ID must be added here. You can add
+    # multiple client IDs to accept tokens from multiple clients.
+    x-google-audiences: "YOUR-CLIENT-ID"
```
Comment (bshaffer) [inline endpoints/getting-started/openapi.yaml]: We need to update the README for the new location of this: 

```diff
-Update the `/auth/info/googleidtoken`'s audiences in openapi.yaml with your client ID.
+Update `google_id_token: x-google-audiences` in openapi.yaml with your client ID.
```

Comment (liminw) [inline endpoints/getting-started/openapi.yaml]: Thanks for the review. Updated README.

**Comment by liminw** [comment]
Gently ping...

**Comment by bshaffer** [comment]
Thank you! It was very gentle.

**Pull request merged**